crates/typos-dict/assets/allowed.csv

```diff
@@ -5,3 +5,4 @@ referer,http header field
 deques,noun
 dequeues,verb
 ons,so `add-ons` works
+accreting,verb
```

Comment:
Could you provide more detail, like "verb of accrete"?

While I was similarly brief with `deques` and `dequeues`, that was more in context of each other (before we corrected one to the other)